app/components/message_attachments/attachment_footer.js

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import {Image, Text, View, Platform} from 'react-native';
+import PropTypes from 'prop-types';
+import truncate from 'lodash/truncate';
+
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+import {MAX_ATTACHMENT_FOOTER_LENGTH} from 'app/constants/attachment';
+
+export default class AttachmentFooter extends PureComponent {
+    static propTypes = {
+        text: PropTypes.string,
+        icon: PropTypes.string,
+        theme: PropTypes.object.isRequired,
+    };
+
+    render() {
+        const {
+            text,
+            icon,
+            theme,
+        } = this.props;
+
+        if (!text) {
+            return null;
+        }
+
+        const style = getStyleSheet(theme);
+
+        return (
+            <View style={style.container}>
+                {Boolean(icon) &&
+                <Image
+                    source={{uri: icon}}
+                    key='footer_icon'
+                    style={style.icon}
+                />
+                }
+                {Boolean(text) &&
```

Comment:
you are already checking for text on line 26, this is not needed here

Reply:
@enahum Good catch 👍 Done.